slidez-core/src/main/resources/db/migration/V9__create_poll_table.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE IF NOT EXISTS poll
+(
+    id                     UUID PRIMARY KEY,
+    interactive_element_id UUID         NOT NULL,
+    title                  VARCHAR(255) NOT NULL,
+    is_multi               BOOLEAN      NOT NULL,
+    is_template            BOOLEAN      NOT NULL,
+    FOREIGN KEY (interactive_element_id) REFERENCES interactive_element (id) ON DELETE CASCADE ON UPDATE CASCADE
+);
```

Comment:
We must not change old migration scripts. 
Or if we have decision to delete all intermediate migration scripts we should combine them all to one script number #1.

Reply:
There will be code quality check and think it will be a little weird if we have 13 scripts that create databasde, then 14-th that deletes everything and then another 13 scripts that create database again